src/core/src/op/gelu.cpp

```diff
@@ -2,41 +2,41 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "ngraph/op/gelu.hpp"
+#include "openvino/op/gelu.hpp"
 
 #include <cmath>
-#include <ngraph/validation_util.hpp>
 
+#include "element_visitor.hpp"
 #include "itt.hpp"
+#include "openvino/core/type.hpp"
 #include "openvino/reference/gelu.hpp"
 
-using namespace std;
-using namespace ngraph;
-
-// ------------------------------ V0 ------------------------------
-op::v0::Gelu::Gelu() : UnaryElementwiseArithmetic() {}
+namespace ov {
+namespace op {
+namespace v0 {
+Gelu::Gelu() : UnaryElementwiseArithmetic() {}
 
-op::v0::Gelu::Gelu(const Output<Node>& data) : UnaryElementwiseArithmetic(data) {
+Gelu::Gelu(const Output<Node>& data) : UnaryElementwiseArithmetic(data) {
     constructor_validate_and_infer_types();
 }
 
-bool op::v0::Gelu::visit_attributes(AttributeVisitor& visitor) {
+bool Gelu::visit_attributes(AttributeVisitor& visitor) {
```

Comment:
This could be removed as is  same as base class?

Reply:
Indeed .. I'll fix it.